src/uwtools/tests/cli/test_atparse_to_jinja2.py

```diff
@@ -0,0 +1,53 @@
+# pylint: disable=duplicate-code,missing-function-docstring
+"""
+Tests for the atparse-to-jinja2 CLI
+"""
+
+from types import SimpleNamespace as ns
+from unittest.mock import patch
+
+import pytest
+from pytest import raises
+
+from uwtools.cli import atparse_to_jinja2
+from uwtools.tests.support import fixture_path
+
+# NB: Ensure that at least one test exercises both short and long forms of each
+#     CLI switch.
+
+
+@pytest.mark.parametrize("sw", [ns(i="-i"), ns(i="--input-template")])
```

Comment:
I know this wasn't the takeaway of this particular comment, but TIL about the few diffs between `SimpleNamespace` instead of `argparse.Namespace`. (https://github.com/python/cpython/issues/83257)